src/components/reusable/card/card.scss

```diff
@@ -0,0 +1,34 @@
+@use '../../../common/scss/global.scss';
+@use '../../../common/scss/mixins.scss';
+@use '~@kyndryl-design-system/shidoka-foundation/scss/mixins/typography.scss';
+
+:host {
+  display: inline-block;
+  width: 264px;
+}
+.card {
+  &-wrapper {
+    position: relative;
+    padding: 24px;
+    background-color: var(--kd-color-background-ui-default);
+    outline: 1px solid var(--kd-color-border-light);
+    border-radius: 8px;
+    height: 100%;
+  }
+  &-wrapper-clickable {
+    position: relative;
+    display: inline-block;
+    color: unset;
+    width: inherit;
+    text-decoration: none;
+    background-color: var(--kd-color-background-ui-default);
+    outline: 1px solid var(--kd-color-border-ui);
+    border-radius: 8px;
+    box-shadow: 0px 1px 2px 0px rgba(61, 60, 60, 0.25);
+    padding: 24px;
+    transition: outline 200ms ease-out;
+    &:hover {
+      outline: 2px solid var(--kd-color-border-ui-hover);
+    }
```

Comment:
Hover needs Elevation level 2

Reply:
ok will use elevation here. Transition is there : ` transition: outline 200ms ease-out;`

Reply:
Hover need elevation level 1 only as per figma for clickable version.